src/test/FreeC/Test/Environment.hs

```diff
@@ -176,7 +180,14 @@ defineTestFunc' partial nameStr arity typeStr = do
 --
 --   Returns the Coq identifier assigned to the function.
 definePartialTestFunc :: String -> Int -> String -> Converter String
-definePartialTestFunc = defineTestFunc' True
+definePartialTestFunc nameStr arity =
+  defineTestFunc' True (replicate arity False) nameStr arity
+
+-- | Like 'defineTestFunc' but also allows to mark arguments as strict in the
+--   first argument.
+defineStrictTestFunc :: [Bool] -> String -> String -> Converter String
+defineStrictTestFunc areStrict nameStr =
```

Comment:
I would prefer the order of arguments to be swapped such that the function is consistent with how you usually write a function declaration in IR, i.e, `<name> <args> :: <type>`.

```suggestion
defineStrictTestFunc :: String -> [Bool] -> String -> Converter String
defineStrictTestFunc nameStr areStrict =
```